src/Akka.Persistence.Sql/Query/Dao/BaseByteReadArrayJournalDao.cs

```diff
@@ -28,7 +28,7 @@ namespace Akka.Persistence.Sql.Query.Dao
 {
     public abstract class BaseByteReadArrayJournalDao : BaseJournalDaoWithReadMessages, IReadJournalDao
     {
-        private readonly Flow<JournalRow, Try<(IPersistentRepresentation, IImmutableSet<string>, long)>, NotUsed> _deserializeFlow;
```

Comment:
Good call on moving from immutable sets to arrays - this will eliminate any additional allocations when we're trying to pass tag collections into the `EventEnvelope`.